src/api/routers/participants/participantsRouter.ts

```diff
@@ -11,7 +11,7 @@ import {
   ParticipantDTO,
   ParticipantStatus,
 } from '../../entities/Participant';
-import { UserDTO, UserRole } from '../../entities/User';
+import { JobFunction, UserDTO } from '../../entities/User';
```

Comment:
Would we want to make this variable name UserJobFunction since it was UserRole before? And the change is Role -> JobFunction?

Reply:
yeah I was on the fence about it since I figured `JobFunction` would be descriptive enough, but have made the change since you mentioned it.